test/new_relic/manticore/instrumentation_test.rb

```diff
@@ -129,11 +112,11 @@ class InstrumentationTest < Minitest::Test
         end
       end
 
-      describe "with cross app tracking enabled" do
+      describe "with cross app tracking disabled" do
```

Comment:
```suggestion
      describe "with distributed tracing enabled" do
```
as the headers are only added if that is enabled